README.md

```diff
@@ -20,11 +20,26 @@ Publish the package on the `npm` registry.
 
 ## Configuration
 
-For each plugin, the `npm` authentication token has to be configured with the environment variable `NPM_TOKEN`.
+### Environment variables
 
-All the plugins are based on `npm` and will use the configuration from `.npmrc`. Any parameter returned by `npm config list` will be used by each plugin.
+The `npm` authentication configuration is **required** and can be set via environment variables.
 
-The registry and dist-tag can be configured in the `package.json` and will take precedence on the configuration in `.npmrc`:
+Both the [token](https://docs.npmjs.com/getting-started/working_with_tokens) and the legacy (`username`, `password` and `email`) authentication are supported. It is recommended to use the [token](https://docs.npmjs.com/getting-started/working_with_tokens) authentication. The legacy authentication is supported as some npm private registry do not suport the token authentication. 
```

Comment:
maybe instead of 

> some npm private registry

it would be better to list the one we know? I think it’s only Artifactory right now? Maybe sth like this?

> The legacy authentication is supported as the alternative npm registry [Artifactory](https://www.jfrog.com/open-source/#os-arti) only supports that form of authentication at this point.

Reply:
There is also https://github.com/npm/npm-registry-couchapp.

Reply:
how about tthis

> The legacy authentication is supported as the alternative npm registries [Artifactory](https://www.jfrog.com/open-source/#os-arti) and [npm-registry-couchapp](https://github.com/npm/npm-registry-couchapp) only supports that form of authentication at this point.

Reply:
I made the change